src/components/dropdown-button.jsx

```diff
@@ -54,7 +54,10 @@ const DropdownButton = ({ children, label, className }) => {
   }, [size]);
 
   return (
-    <div className="dropdown-container">
+    <div
```

Comment:
Noticed that the carets are not visible for level 2 and 3 buttons. Also the button margin has changed, don't know if that was on purpose or not
![image](https://user-images.githubusercontent.com/3329765/98566682-fa0cf200-22a6-11eb-87b6-fc598177f7df.png)
![image](https://user-images.githubusercontent.com/3329765/98566793-1b6dde00-22a7-11eb-9f6f-67ed49e94ef3.png)

Reply:
Fixed the arrow colour.
I removed the margin because I didn't know why it was there and we had to fight against it in the places where we want to use it anyway (for example, the query build alignment issue was mainly that, and now we have a `margin-right: 0` against it)

Reply:
bottom margin back now, and overriding it in the header